docs/sphinx/user_guide/cook_book/reduction.rst

```diff
@@ -52,15 +52,15 @@ RAJA uses policy types to specify how things are implemented.
 
 The forall *execution policy* specifies how the loop is run by the ``RAJA::forall`` method. The following discussion includes examples of several other RAJA execution policies that could be applied.
 For example ``RAJA::seq_exec`` runs a C-style for loop sequentially on a CPU. The
-``RAJA::cuda_exec_rec_for_reduce<256>`` runs the loop as a CUDA GPU kernel with
+``RAJA::cuda_exec_reduce_default<256>`` runs the loop as a CUDA GPU kernel with
```

Comment:
Should this say something about using this policy for a kernel with a reduction operation in it?

Reply:
That's a reasonable thing to expect. How about `cuda_exec_tuned_for_reduce`

Reply:
What about `cuda_exec_use_with_reductions_because_jason_said_so`? 😄 

Seriously, let's go with names that are simple and easy to understand. I like `cuda_exec_with_reduce`.

Reply:
Ok, I'll go with that.